query_execution/WorkOrdersContainer.hpp

```diff
@@ -311,16 +311,17 @@ class WorkOrdersContainer {
     }
 
     inline void addWorkOrder(WorkOrder *workorder) {
-      workorders_.push(workorder);
+      workorders_.emplace(workorder);
```

Comment:
Yes, either `emplace` or `emplace_back` creates a `unique_ptr` to contain `workorder`, which is created by `new`.

This is the same as the first example from the best answer by `Angew` in your given link.

@craig-chasseur Any comments?